easybuild/easyconfigs/g/genoPlotR/genoPlotR-0.8.11-foss-2020b.eb

```diff
@@ -0,0 +1,28 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Denis Kristak
+easyblock = 'RPackage'
+
+name = 'genoPlotR'
+version = '0.8.11'
+
+homepage = 'https://cran.r-project.org/web/packages/genoPlotR/index.html'
+description = """Draws gene or genome maps and comparisons between these, in a publication-grade manner.
+Starting from simple, common files, it will draw postscript or PDF files that can be sent as such to journals.
+"""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+source_urls = ['https://cran.r-project.org/src/contrib/']
+sources = ['%s_%s.tar.gz' % (name, version)]
+checksums = ['f127f7fe8b19c899ecfdf98bf69d2e18926afb593a72fc40097acca66d401607']
+
+dependencies = [
+    ('R', '4.0.3'),
```

Comment:
Since this only depends on `R`, we should probably just add this as an extension to `R` itself?

I would prefer doing that only for the latest `R` version we have though (4.1.2 with `foss/2021b`)...

Reply:
@deniskristak I've added `genoPlotR` to `R` 4.1.2 in https://github.com/easybuilders/easybuild-easyconfigs/pull/14433